.pre-commit-config.yaml

```diff
@@ -42,8 +42,5 @@ repos:
     hooks:
       - id: mypy
         pass_filenames: false
-        args: [ ., --strict, --explicit-package-bases,
-                  --disable-error-code, import-untyped,
-                  --disable-error-code, import-not-found,
-                  --disable-error-code, no-untyped-call,
-                  --disable-error-code, type-arg ]
+        args: [ . ]
```

Comment:
If you add `packages`, then I think you can remove this. At least then it works locally for me by just specifying `mypy` on the command line.